src/app/content/highlights/components/CardWrapper.tsx

```diff
@@ -264,8 +264,30 @@ const Wrapper = ({highlights, className, container, highlighter, dispatch}: Wrap
   </div>;
 };
 
+function extractTextFromHTML(html: string): string {
```

Comment:
We already have a [stripHTML utility](https://github.com/openstax/rex-web/blob/main/src/app/utils.ts#L155) that should work for this.